src/ripple/shamap/impl/SHAMapSync.cpp

```diff
@@ -386,10 +386,10 @@ SHAMap::getMissingNodes(int max, SHAMapSyncFilter* filter)
             if (mn.stack_.empty() && ! mn.resumes_.empty())
             {
                 // Recheck nodes we could not finish before
-                for (auto& r : mn.resumes_)
-                    if (! r.first->isFullBelow (mn.generation_))
+                for (auto& [innerNode, nodeId] : mn.resumes_)
```

Comment:
auto const&

Reply:
Fixed in a225b62e5 `Add const to some structured bindings`